### DISCUSSION TITLE
텍스트 파일 읽기

### DISCUSSION THREAD

**Discussion by natural-jogi** [naverz/zepeto-studio-kor#378] [5. Q&A]
C# 문법에서는 StreamReader를 이용하면 되는것으로 알고 있는데, 제페토 스크립트에서 .txt 파일을 읽어올 수 있는 방법이 있는 지 궁금합니다.

**Comment by pumfit** [comment]
유니티에서 제공되는 TextAsset을 사용해 .txt 파일을 읽을 수 있습니다.
아래와 같이 코드를 작성하시고 
```import { Resources, TextAsset } from 'UnityEngine'
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class TextAssetExample extends ZepetoScriptBehaviour {

    Start() {    
        let textAsset :TextAsset = Resources.Load("example") as TextAsset;
        console.log(`${textAsset.text}`);
    }

}
```

텍스트 파일을 리소스 폴더에서 불러와 내용을 확인할 수 있습니다!

![image](https://user-images.githubusercontent.com/46295539/153971836-3604646a-4aee-4f5c-bf7f-f995174e8026.png)


![image](https://user-images.githubusercontent.com/46295539/153828724-b5f2d9c6-0dc0-489e-a261-b42a970413bb.png)
